src/autocomplete/autocomplete-types.ts

```diff
@@ -154,6 +155,7 @@ export interface AutocompleteData<
     ignoredTokens: Set<number>;
     rulesToVisit: Set<number>;
     enrichAutocompleteResult: EnrichAutocompleteResult<A>;
+    context?: ParserRuleContext;
```

Comment:
why do we need this? can't see it being used anywhere

Reply:
> why do we need this? can't see it being used anywhere

We use context here: https://github.com/gravity-ui/websql-autocomplete/pull/173/files#diff-038083b7de04219ec7c935fd3387dbc39cb1b97b34e468a0539d4cdd2f58bbc6R80 . It is needed to arrange an entrypoint for parsing query to suggest keywords. If no context is given, all tree will be taken, no matter what you use in `parseTree` function.